releasenotes/notes/tranform-API-da38b196c1c72d44.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    It's now possible to made some transformation to the stored timeseries
+    during the retrieval of the measures. The allowed transformations are
+    absolute and negative.
```

Comment:
It's documented here: https://github.com/gnocchixyz/gnocchi/pull/201/files#diff-3c5a1f8905561bf156b5e5fc97645cb2R120

but if the writting is bad (and I will be not surprise), tell me how I can enhance it.